variables.tf

```diff
@@ -45,3 +45,8 @@ variable "dns_zonal_record_ttl" {
   type        = string
   default     = 60
 }
+
+variable "ably_vpc_service_endpoint_name" {
+  type        = string
```

Comment:
Does this need to be a per-region variable?

Reply:
In short, no. A module resource is declared per region.

Reply:
Ok got it 👍